op-service/txmgr/txmgr.go

```diff
@@ -815,10 +816,14 @@ func (m *SimpleTxManager) queryReceipt(ctx context.Context, txHash common.Hash,
 	}
 
 	m.metr.RecordBaseFee(tip.BaseFee)
-	if tip.ExcessBlobGas != nil {
-		blobFee := eth.CalcBlobFeeDefault(tip)
-		m.metr.RecordBlobBaseFee(blobFee)
+
+	blobFee, err := m.backend.BlobBaseFee(ctx)
+	if err != nil {
+		m.metr.RPCError()
+		m.l.Error("Unable to fetch blob base fee", "err", err)
+		return nil
 	}
+	m.metr.RecordBlobBaseFee(blobFee)
```

Comment:
Arguably we don't even need to return here in case of error. That we even query the blob base fee here and record it is just an optimization to have more fine grained metrics. But if fetching the head above succeeded, there's no reason to abort the receipt querying. The blob base fee is only fetched to record it.

```suggestion
	if blobFee, err := m.backend.BlobBaseFee(ctx); err != nil {
		m.metr.RPCError()
		m.l.Warn("Unable to fetch blob base fee", "err", err)
	} else {
		m.metr.RecordBlobBaseFee(blobFee)
	}
```